<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>orderDetails</class>
 <widget class="QWidget" name="orderDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>订单详情</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.ico</normaloff>logo.ico</iconset>
  </property>
  <widget class="QLabel" name="skuNameBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="statusBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>收货地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="addressBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="orderNoBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>物流：</string>
   </property>
  </widget>
  <widget class="QListView" name="wuLiuListBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>151</y>
     <width>611</width>
     <height>341</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
  </widget>
  <widget class="QLabel" name="wuLiuNameBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wuLiuNoBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>订单号：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
